Viper/Viper/Interactor/CountryInteractorInputProtocol.swift

```diff
@@ -0,0 +1,10 @@
+// CountryInteractorInputProtocol.swift
+// Copyright © Natasha Ananas. All rights reserved.
+
+import Foundation
+
+/// Input Протокол интератора
```

Comment:
```suggestion
/// Input Протокол интератора
```
Или на английском, или на русском. Тут чет вообще даблкилл))